README.md

```diff
@@ -1,416 +1,178 @@
-
-#### Quick Start
-
-0. _(assumed)_ Install missing or out-of-date Node dependencies.\*
-1. _(optional)_ Make changes to `/taccsite_custom/name-of-project/static/name-of-project/css/src` files. †‡
-2. Build static files from source files.\
-    Via shell:
-    1. `npm run build --project=name-of-project` †\
-        or\
-        `npm run build (...) --build-id=optional-identifier` §
-3. _(to debug)_ Review respective files' content in\
-    `/taccsite_custom/name-of-project/static/name-of-project/css/build`. †
-4. "Collect" static files. _See [How to Collect Static Files](#how-to-collect-static-files)._
-5. _(to debug)_ Confirm respective output changed in:\
-    `/taccsite_cms/static/site_cms/css/build`\
-    and/or\
-    `/taccsite_custom/static/name-of-project/css/build` †
-
-<sub>\* The recommended command to install expected dependencies is `npm ci`.</sub>\
-<sub>† Where `name-of-project` is either `core-cms` or a directory from `/taccsite_custom`.‡</sub>\
-<sub>‡ If your custom CMS project is in [Core CMS Custom], you do __not__ need this step.</sub>\
-<sub>§ To commit such changes, see [Changing Custom Resources](#changing-custom-resources).</sub>\
-<sub>‖ A build ID can tag files (e.g. preserved comment in stylesheet).</sub>
-
-#### How to Build Static Files
-
-Certain static files are built __from__ source files __in__ `src` directories __to__ compiled files __in__ `build` directories.
-
-1. Build static resources: \*†
-
-    ```bash
-    npm run build --project=name-of-project
-    ```
-
-<sub>\* You should run these commands in the container __from `/code/`__. _See [Running Commands in Container](#running-commands-in-container)._</sub>\
-<sub>† Where `name-of-project` is either `core-cms` or a directory from `/taccsite_custom`.‡</sub>\
-<sub>‡ If your custom CMS project is in [Core CMS Custom], you do __not__ need this step.</sub>
-
-#### How to Collect Static Files
-
-Whenever static files are changed, the CMS must be manually told to serve them.\*
-
-<!-- TODO: [Automatically perform `collectstatic`](https://stackoverflow.com/q/59339571/11817077) -->
-
-1. [Collect static files][django-static] for Django: \*
-
-    ```bash
+    # To use default "Username" and skip "Email address", press Enter at both prompts.
+    # At "Password" prompts, you may use an easy-to-remember password.
+    npm ci
+    npm run build:css --project="core-cms"
     python manage.py collectstatic --no-input
     ```
 
-<sub>\* You should run these commands in the container __from `/code/`__. _See [Running Commands in Container](#running-commands-in-container)._</sub>
-
-### Changing Custom Resources
-
-If you need to get the latest files into `/taccsite_custom` from [Core CMS Resources]:
-
-1. [How to Change Submodule Branch Commit](https://github.com/TACC/Core-CMS/wiki/How-to-Change-Submodule-Branch-Commit)
+3. Django CMS:
+    1. Open http://0.0.0.0:8000/.
```

Comment:
I'm stepping a little out of my depth here, but should we use `localhost:8000`? Wouldn't the IP depend on local configuration?

Reply:
```suggestion
    1. Open http://localhost:8000/.
```

Yes. This was copied. It was `localhost`. Thanks.

I'll update the repos I copied from too.

Reply:
My"Yes" is to whether we should use `localhost:8000`. I **believe** the IP depends on the local configuration, which is I **think** [is 127.0.0.1 thus localhost](https://github.com/TACC/Core-CMS/blob/main/docker-compose.dev.yml#L8-L10).

Reply:
Wes, you updated the sources:
- https://github.com/TACC/Core-CMS-Resources/blob/docs/readme-improvements/README.md#start-project
- https://github.com/TACC/Core-CMS-Custom#start-project